src/components/pages/Home/Home.tsx

```diff
@@ -32,35 +30,19 @@ export default function Home(props: Props): React.ReactElement {
           return {
             url: firstImageItem.url,
             id: firstImageItem.sys.id,
+            clickUrl: `${locale}${EXHIBIT_URL}/${exhibit.slug}`,
```

Comment:
Well, in this case, I am afraid, we might not see locale used in path in the browser corner with navigation URL displaying, from my point of view it might be a little inconsistent with actual browser URL. What do you think?
<img width="364" alt="Screenshot 2024-12-04 at 23 24 50" src="https://github.com/user-attachments/assets/add36319-0263-4228-88be-96f6306db7c8">

Reply:
I guess it is ok. It looks like this is a default behaviour for next-intl